.semaphore/update_pins.yml

```diff
@@ -0,0 +1,33 @@
+version: v1.0
+name: Trigger Pin Updates
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+
+global_job_config:
+  secrets:
+    # Mount the github SSH secret for pulling private repositories.
+    - name: private-repo
```

Comment:
Is this needed? I am asking since this is a public repo or perhaps the comment needs to be changed?

Reply:
I think it is because it has Marvins github creds which is needed for public and private